templates/utils.js

```diff
@@ -64,7 +88,7 @@ export const withDefaults =
 
       if (receivedType !== expectedType) {
         throw new Error(
-          `Template argument \`${receivedArgName}\` has wrong type. Expecting ${
+          `Template ${callerFile} argument \`${receivedArgName}\` has wrong type. Expecting ${
```

Comment:
<img width="834" alt="Screenshot 2025-06-09 at 4 23 36 PM" src="https://github.com/user-attachments/assets/7b2331ab-9f45-4ceb-a03e-bcfa22fda000" />

Instead of "Expecting..." what if we keep the error similar to typescript? 

"Expected string but received array" ?

Reply:
Thanks, fixed. Also changed a similarly for unexpected argument [ebc4993](https://github.com/scaffold-eth/create-eth/pull/279/commits/ebc49931e7e32bc4160f7a749a1ef1525216d12b)